### ISSUE TITLE
php snippets working perfectly inside short open tags <? ?> but not inside <?php ?>

### ISSUE THREAD

**Issue by genmacg** [Microsoft/vscode#39155]
Whenever I try to insert a user snippet (defined in php.json) inside of short open tags `<? ... ?>` in a .php file everything works fine. All snippets show up as suggestions immediately while typing without any delay.
```
<?
// ... inserting snippet here works fine
?>
```

Whenever I do the same inside of `<?php ... ?>` it takes around 3 seconds for any suggestions to show up and most snippets are missing from those completely.
```
<?php
// ... inserting snippet here doesn't work as expected
?>
```

VScode Version: 1.18.1
OS Version: MacOS Sierra 10.12.6

**Comment by roblourens** [comment]
Do you have any php extensions installed?

> most snippets are missing from those completely.

That's weird, are these your own custom snippets, or from an extension? Is it consistent which ones show up and which don't?

**Comment by genmacg** [comment]
Ok, now I'm either crazy or I had a serious d'oh moment. I went ahead and disabled all extensions and the snippets started working again. So far so good. I started enabling my extensions again one by one with reloading VScode in between to see which extension might be the culprit. But none was. I was able to enable all of them again and my snippets seem to work fine again. 

What remains though is the difference in performance. Suggestions show up immediately while typing inside short open tags, otherwise, it takes around 2 seconds. The delay disappeared when I disabled PHP IntelliSense (2.0.1).
